app/controllers/api/integrations_controller.rb

```diff
@@ -109,6 +110,10 @@ def show
     end
   end
 
+  def instantiate_presenter
```

Comment:
I do not see the benefit of having it as a method

Reply:
In order to have it also in the `edit` method: https://github.com/3scale/porta/pull/1368/files#diff-03ca76903039d35f4565cfda67324befR26
, when rendering :show.